app/controllers/evaluator_submission_assignments_controller.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+class EvaluatorSubmissionAssignmentsController < ApplicationController
+  before_action :set_challenge_and_phase
+  before_action :set_evaluator, only: [:index]
+  before_action :set_assignment, only: [:update]
+
+  def index
+    @evaluator_assignments = @phase.evaluator_submission_assignments.
+      includes(:submission, :evaluation).
+      where(user_id: @evaluator.id)
+    @assigned_submissions = @evaluator_assignments.
+      where(status: %i[completed in_progress not_started recused]).
+      ordered_by_status
+    @unassigned_submissions = @evaluator_assignments.
+      where(status: %i[unassigned recused_unassigned]).
+      ordered_by_status
+    @submissions_count = @assigned_submissions.group('evaluator_submission_assignments.status').count
+  end
+
+  # update only the status of the evaluation submission assignment to unassign or reassign an evaluator
+  def update
+    new_status = params[:status]&.to_sym
+
+    if update_assignment_status(new_status)
+      handle_successful_update(new_status)
+    else
+      handle_failed_update(new_status)
+    end
+  end
+
+  private
+
+  def set_challenge_and_phase
+    @phase = Phase.where(challenge: current_user.challenge_manager_challenges).find(params[:phase_id])
+    @challenge = @phase.challenge
+  end
+
+  def set_evaluator
+    @evaluator = params[:evaluator_id] ? User.find(params[:evaluator_id]) : current_user
+  end
+
+  def set_assignment
+    @assignment = EvaluatorSubmissionAssignment.find_by!(
+      user_id: params[:evaluator_id],
+      submission_id: params[:submission_id]
+    )
```

Comment:
I suppose this is why you added the custom route, but I think it's better to use the ID param from the normal update route. `PATH /phases/:path_id/evaluator_submission_assignments/:id` which is the id for the EvaluatorSubmissionAssignment record. You still want to authorize the current_user's access, which you can do by going through the `@phase` association since `@phase` is already restricted to the current_user. This ensures that a user can't unassign/reassign evaluators for some random challenge phase they're not associated with. Rails lets you do this
```suggestion
    @assignment = EvaluatorSubmissionAssignment.where(evaluator: @phase.evaluators).find(params[:id])
```